atbash-cipher/example.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Atbash {
+
+    private static final int GROUP_SIZE = 5;
+    private static final String PLAIN = "abcdefghijklmnopqrstuvwxyz";
+    private static final String CIPHER = "zyxwvutsrqponmlkjihgfedcba";
+
+    public static String encode(String input) {
+        String encoded = stripInvalidCharacters(input).toLowerCase();
+        String cyphered = "";
+
+        for (char c : encoded.toCharArray()) {
+            cyphered += applyCipher(c);
+        }
+
+        return splitIntoFiveLetterWords(cyphered);
+    }
+
+    private static String stripInvalidCharacters(String input) {
+        String filteredValue = "";
+
+        for (char c : input.toCharArray()) {
+            if (Character.isLetterOrDigit(c)) {
+                filteredValue += c;
+            }
+        }
+
+        return filteredValue;
+    }
+
+    private static char applyCipher(char input) {
+        int idx = PLAIN.indexOf(input);
+
+        return idx >= 0 ? CIPHER.toCharArray()[idx] : input;
```

Comment:
Could extract a constant of `CIPHER.toCharArray()` --- not sure if the compiler would optimize this to avoid having to run this conversion every time.